XCHammerAssets/Hammer.bzl

```diff
@@ -1,9 +1,9 @@
 load(
-    "@build_bazel_rules_apple//apple/bundling:entitlements.bzl",
+    "@build_bazel_rules_apple//apple/internal:entitlement_rules.bzl",
```

Comment:
this isn't great since were depending on private api now so we should upstream a target for this or rely on internal rules and potentially vendor them

Reply:
Consider the maintainability tradeoff of that. I’d suggest making this rule public in rules_apple since it’s a useful utility 🤙.

Reply:
Yeah I filed a ticket to track resolving this https://github.com/pinterest/xchammer/issues/154